plugins/callback/foreman.py

```diff
@@ -250,10 +263,43 @@ def send_facts(self):
 
             self._send_data('facts', host, facts)
 
-    def send_reports(self, stats):
+    def send_reports_proxy_host_report(self, stats):
+        """
+        Send reports to Foreman Smart Proxy running Host Reports
+        plugin. The format is native Ansible report without any
+        changes.
+        """
+        for host in stats.processed.keys():
+            total = stats.summarize(host)
+            report = {
+                "host": host,
+                "reported_at": get_now(),
+                "uuid_XXX": "XXXXXXXXXXXXXXXXXXXXXXXXXXX",
```

Comment:
I am not sure if there is an unique UUID per report per host, we might need to generate one.